workshops/scipy2023/README.md

```diff
@@ -19,6 +19,16 @@ See instructions to set up the environment for running the tutorial material [he
 For the live tutorial, we will be using the SciPy 2023 Nebari JupyterHub. Instructions for the live
 tutorial will be added soon.
 
+## Github Codespaces
+
+This tutorial is available to run within github codespaces with the conda environment specification
+in the [`conda-lock.yml`](../../conda/conda-lock.yml) file.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/xarray-contrib/xarray-tutorial/tree/main?devcontainer_path=.devcontainer%2Fscipy2023%2Fdevcontainer.json)
```

Comment:
This badge should work once this PR is merged. To try out, try below

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/lsetiawan/xarray-tutorial/tree/add_codespaces?devcontainer_path=.devcontainer%2Fscipy2023%2Fdevcontainer.json)

Reply:
Once terminal is open, run `jupyter lab`. There must be a way in VS Code to automate this... will have to figure out how.

Reply:
Let's add this badge to README.md too

Reply:
That one is for that particular branch though, which will get deleted once this PR is merged. There's already a badge in there for the actual thing after merging.

Reply:
You can see the rendered README here with the button: https://xarray-contrib-xarray-tutorial-preview-pr-184.surge.sh/workshops/scipy2023/README.html

Reply:
Sorry I meant for the top level Readme for the general user, not just scipy attendees. Assuming that makes sense in the context of how this is set up...

Reply:
Ah gotcha. This setup currently is very specific for the scipy2023. We can make a default configuration in separate PR.